tests/System.Text.Http.Tests/GivenAnHttpHeaders.cs

```diff
@@ -54,11 +54,13 @@ public void Its_enumerator_Current_returns_the_same_item_until_MoveNext_gets_cal
             enumerator.MoveNext();
 
             var current = enumerator.Current;
-            current.Should().Be(enumerator.Current);
+            //TODO: no spans on the heap
```

Comment:
@shiftylogic, I don't know what we should do about this. It is part of a skipped test already. Thoughts?